_includes/samples/DataProviders/DnsOwnership.sol

```diff
@@ -13,42 +13,49 @@ contract DnsOwnershipChainlink is ChainlinkClient, ConfirmedOwner {
     using Chainlink for Chainlink.Request;
     
     uint256 oraclePayment;
+    address private oracle;
+    bytes32 private jobId;
     bool public proof;
     /**
      * Network: Kovan
      * Oracle: 
      *      Name:           LinkPool
      *      Listing URL:    https://market.link/nodes/323602b9-3831-4f8d-a66b-3fb7531649eb?network=42
-     *      Address:        0x56dd6586DB0D08c6Ce7B2f2805af28616E082455
+     *      Address:        0xff07c97631ff3bab5e5e5660cdf47aded8d4d4fd
      * Job: 
      *      Name:           DNS Record Check
      *      Listing URL:    https://market.link/jobs/bf1a410f-ce93-497d-83ac-e63fed9d83bd?network=42
      *      ID:             791bd73c8a1349859f09b1cb87304f71
      *      Fee:            0.1 LINK
      */
-    constructor(uint256 _oraclePayment) ConfirmedOwner(msg.sender) {
+    constructor() ConfirmedOwner(msg.sender) {
       setPublicChainlinkToken();
-      oraclePayment = _oraclePayment;
+      oracle = 0xfF07C97631Ff3bAb5e5e5660Cdf47AdEd8D4d4Fd;
```

Comment:
is this oracle address correct? looking at the job on [market.link](https://market.link/jobs/bf1a410f-ce93-497d-83ac-e63fed9d83bd) it has another listed oracle contract